Modules/mathmodule.c

```diff
@@ -2016,6 +2016,79 @@ math_factorial(PyObject *module, PyObject *arg)
 }
 
 
+/*[clinic input]
+math.lcm
+
+	x as a: object
+	y as b: object
+    /
+least common multiple of x and y
+[clinic start generated code]*/
+
+static PyObject*
+math_lcm(PyObject *module, PyObject *a, PyObject *b)
+{
+    PyObject *g, *m, *f, *ab;
+	
+    a = PyNumber_Index(a);
+	
```

Comment:
Remove this empty line. Empty lines should be used to separate related groups of lines.